segments-explainer/SEGMENTS-EXPLAINER.md

```diff
@@ -8,32 +11,35 @@
 ## Related explainers:
 | Name | Link |
 |------|------|
-| Screen Fold API | [Explainer](https://github.com/SamsungInternet/Explainers/blob/master/Foldables/FoldState.md), [Unofficial Draft Spec](https://w3c.github.io/screen-fold/) |
+| Device Posture API | [Explainer](https://github.com/w3c/device-posture), [Working Draft Spec](https://www.w3.org/TR/device-posture/) |
 | Multi-Screen Window Placement API | [Explainer](https://github.com/webscreens/window-placement/blob/master/EXPLAINER.md), [Draft Community Group Report](https://webscreens.github.io/window-placement/) |
-| Visual Viewport API | [Draft Community Group Report](https://wicg.github.io/visual-viewport/), [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Visual_Viewport_API) |
+| Visual Viewport API | [Draft CSSOM View Spec](https://drafts.csswg.org/cssom-view/#visualViewport), [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Visual_Viewport_API) |
 
 ## Motivation:
-Web developers targeting foldable devices want to be able to effectively lay out the content in a window that spans multiple displays. However, the web platform does not yet provide the necessary primitives for building layouts that are optimized for foldable experiences.
+Web developers targeting foldable devices want to be able to effectively lay out the content in a window that spans multiple displays or across the fold of device with a foldable screen. However, the web platform does not yet provide the necessary primitives for building layouts that are optimized for foldable experiences.
 Developers may be able to solve this by taking a hard dependency on a specific device hardware parameters - an approach that is fragile, not scalable, and requires work duplication for each new device.
 
 ### Current problems:
 More specific challenges we've heard from our internal product teams that were exploring building experiences for this emerging classes of devices include:
 
-- *Hardware differences*: Devices could be seamless (e.g. Samsung Galaxy Fold) or have a seam (e.g. [Microsoft Surface Duo](https://www.microsoft.com/en-us/surface/devices/surface-duo) or ZTE Axon M). In the latter case developers might want to take it into account or intentionally ignore depending on scenario;
+- *Hardware differences*: Devices could be seamless (e.g. [Samsung Galaxy Fold](https://www.samsung.com/levant/smartphones/galaxy-fold/), [Asus Zenbook Fold](https://www.asus.com/laptops/for-home/zenbook/zenbook-17-fold-oled-ux9702/)) or have a seam (e.g. [Microsoft Surface Duo 2](https://www.microsoft.com/en-us/d/surface-duo-2/9408kgxp4xjl), [Asus Zenbook Duo](https://www.asus.com/us/laptops/for-home/zenbook/asus-zenbook-duo-2024-ux8406/) or ZTE Axon M). In the former case developers might want to take it into account or intentionally ignore depending on scenario;
```

Comment:
Does't "developers might want to take it into account or intentionally ignore depending on scenario" apply to either case?

Reply:
I think you are right.